region/tourian/main/Tourian Escape Room 4.json

```
@@ -567,6 +551,110 @@
+      "requires": [
+        {"notable": "Reverse Tourian Escape Room 4 Bootless Walljumpless Space Jump"},
+        "SpaceJump",
+        "canTrickyJump",
+        {"acidFrames": 90}
+      ],
+      "note": [
+        "Climbing the shaft with Space Jump is slower than other methods, so it is necessary to move quickly in order to minimize acid damage.",
+        "After reaching the top, fall down the right side to land on the platform below the door, taking a dip in acid before jumping into the door."
+      ]
+    },
+    {
+      "link": [3, 1],
+      "name": "Reverse Tourian Escape Room 4 Bootless Walljumpless Space Jump (Short Acid Dip)",
+      "requires": [
+        {"notable": "Reverse Tourian Escape Room 4 Bootless Walljumpless Space Jump"},
+        "canPreciseSpaceJump",
+        "canTrickyJump",
+        {"acidFrames": 50}
+      ],
+      "note": [
+        "Climbing the shaft with Space Jump is slower than other methods, so it is necessary to move quickly in order to minimize acid damage.",
+        "With each Space Jump, release jump early rather than doing a full-height jump, in order to be able to Space Jump again more quickly.",
+        "After reaching the top, fall down on the right side to land on the platform below the door, taking a brief dip in acid before jumping into the door."
+      ]
+    },
+    {
+      "link": [3, 1],
+      "name": "Reverse Tourian Escape Room 4 Bootless Walljumpless Space Jump (Plasma)",
+      "requires": [
+        {"notable": "Reverse Tourian Escape Room 4 Bootless Walljumpless Space Jump"},
+        "canPreciseSpaceJump",
+        "canTrickyJump",
+        "Plasma",
+        {"or": [
+          {"enemyDamage": {
+            "enemy": "Tourian Space Pirate (all)",
+            "type": "contact",
+            "hits": 1
+          }},
+          {"and": [
+            "canHitbox",
+            "canInsaneJump"
+          ]}  
+        ]}
+      ],
+      "note": [
+        "Climbing the shaft with Space Jump is slower than other methods, so it is necessary to move quickly in order to minimize acid damage.",
+        "With each Space Jump, release jump early rather than doing a full-height jump, in order to be able to Space Jump again more quickly.",
+        "After reaching the top, fall down on the right side to land on the platform above the door.",
+        "Then press against the left wall and pass through the last Pirate while firing a Plasma shot down to open the door;",
+        "with precise timing, damage from the Pirate can be avoided by hitboxing through it.",
+        "Hold left to land directly in the doorway.",
+        "If done quickly enough, it is possible to avoid any acid damage."
+      ]
+    },
+    {
+      "link": [3, 1],
+      "name": "Reverse Tourian Escape Room 4 Bootless Walljumpless Space Jump (Super)",
+      "requires": [
+        {"notable": "Reverse Tourian Escape Room 4 Bootless Walljumpless Space Jump"},
+        "canPreciseSpaceJump",
```

```suggestion
        "canPreciseSpaceJump",
        "canTrickyJump",
        "canAutoCancelWeapon",
```
Should precise space jump just require tricky jump?

---

I think of `canPreciseSpaceJump` as a higher movement tech that kind of requires tricky jumps by definition, so yeah it would probably make sense for it to have canTrickyJump as a dependency yeah; then redundant `canTrickyJump` requirements could be removed from the other strats.

The auto cancel is not required here; it helps simplify the sequence of inputs, but I think it would still be ok in this difficulty without it, so doesn't need to be a logical requirement